src/test/java/io/jenkins/plugins/analysis/core/util/LogHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.analysis.core.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Report;
+
+import hudson.model.TaskListener;
+
+import static io.jenkins.plugins.analysis.core.assertions.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link LogHandlerTest}.
+ *
+ * @author Andreas Neumeier
+ */
+class LogHandlerTest {
+    private static final String LOG_HANDLER_NAME = "TestHandler";
+    private static final String MESSAGE = "TestMessage";
+    private static final TaskListener TASK_LISTENER = mock(TaskListener.class);
```

Comment:
Do not use static fields for stubs as this might not work if the tests are executed in parallel.

Reply:
@uhafner Is it ok to leaf it as private final? 

Nevertheless I moved the stub into a method to save lines, because it is required in multiple tests.

Reply:
Yes, this is how I actually want to have it :-)